src/app/item-review/destiny-tracker.service.ts

```diff
@@ -44,6 +44,8 @@ export interface DtrUserReview {
   isIgnored?: boolean;
   selectedPerks: number[];
   attachedMods: number[];
+  mode: number;  // play mode; corresponds to DestinyActivityModeType
+  sandbox: number; // sandbox season (1 was the first, 2 is the March 2018 "go fast" update)
```

Comment:
You can type this as `1 | 2`

Reply:
Doesn't that constrain it to being 1 or 2?

I'd rather not have to push a new app version because the API incremented the sandbox.